docs/src/_getting-started/part1.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Tutorial Part 1 - Extract Data
+description: Part 1 - If you're ready to get started with Meltano and run an EL[T] pipeline with a data source and destination of your choosing, you've come to the right place!
+layout: doc
+weight: 3
+---
+Let’s learn by example.
+
+Throughout this tutorial, we’ll walk you through the creation of a end-to-end modern E(t)LT stack.
+
+We're going to take data from a "source", namely GitLab, and extract a list of projects, the authors, messages and so on.
```

Comment:
Couple of questions on this:

* do you think it's a problem to have a source that a user will need to have an account on? I think GitLab may actually require a credit card now to signup (I could be wrong though). We could have a source that doesnt' require authtentication
* Should we prefer GitHub to GitLab? The bet is that folks will be more likely to complete the tutorial if they have an existing GH account.

Reply:
Mh I just signed up for GitLab, so it doesn't. But yes I do think it's a problem, but I couldn't think of a better alternative at the moment. GitHub could be a better option. I definitely think it should be an external source, not something local, otherwise it's hard to convince people of the feeling of doing actual data integration. So yes we could switch over.